### ISSUE TITLE
Need Phi-3-vision batch size > 1

### ISSUE THREAD

**Issue by hommayushi3** [microsoft/Phi-3CookBook#158]
### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ x ] feature request
- [ x ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Details
I am currently running a finetuned Phi-3-vision through VLLM, but my throughput is capped at the speed of a single threaded inference, despite running concurrent calls. I figured out that continuous batching is correctly batching the calls, but the batch inference of the Phi-3-vision model is linear in time with the batch size.

Investigating further, I noticed in your training scripts it mentions that [batch size > 1 is not supported](https://github.com/microsoft/Phi-3CookBook/blob/1d32acecff1dd22d5b2c7b8e5f8daaa0da913d1d/code/04.Finetuning/vision_finetuning/finetune_hf_trainer_docvqa.py#L145). I also noticed this [PR/Issue](https://huggingface.co/microsoft/Phi-3-vision-128k-instruct/discussions/32), which has an potential fix, though it seems unverified.

My questions are:
1. Is this issue with batched requests fixed with Phi-3.5-vision? (I see batch size = 64 in [this training script](https://github.com/microsoft/Phi-3CookBook/blob/main/md/04.Fine-tuning/FineTuning_Vision.md#docvqa-note-phi-3-vision))
2. If not, is there a plan/timeline to allow for batched training/inference?

**Comment by leestott** [comment]
@chenrocks can you please comment on the new sample

**Comment by ChenRocks** [comment]
Thanks for your inquiry.

> My questions are:
> 
> 1. Is this issue with batched requests fixed with Phi-3.5-vision? (I see batch size = 64 in [this training script](https://github.com/microsoft/Phi-3CookBook/blob/main/md/04.Fine-tuning/FineTuning_Vision.md#docvqa-note-phi-3-vision))

Here an effective batch_size of 64 is achieved via gradient accumulation. Each forward/backward pass still runs on batch_size == 1.

> 2. If not, is there a plan/timeline to allow for batched training/inference?

The UCF-101 example demonstrates batch_size_per_gpu > 1. I'd rather keep other examples using batch_size_per_gpu == 1 for simplicity. After all, the scripts are for learning purposes rather than optimized for speed.
